org.kde.qmlkonsole.json

```diff
@@ -0,0 +1,75 @@
+{
+    "id": "org.kde.qmlkonsole",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-22.08",
+    "sdk": "org.kde.Sdk",
+    "command": "qmlkonsole",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.freedesktop.Flatpak"
```

Comment:
this dbus permission is why the build fails.

Reply:
There is no more fine-grained permission to allow spawning on the host, right? For a terminal app that is pretty much required to be useful.